docs/cli/cmdline/script/info.md

```diff
@@ -0,0 +1,49 @@
+---
+title: terramate script info - Command
+description: The script info command gives you information about a script
+
+prev:
+  text: 'Run'
+  link: '/cli/cmdline/run'
+
+next:
+  text: 'Script List'
+  link: '/cli/cmdline/script/list'
+---
+
+# Script Info
+
+**Note:** This is an experimental command that is likely subject to change in the future.
+
+The `script info LABEL...` command lists details about all script definitions matching the given LABELs (see [./run](run) command for details about matching labels). The information provided by `script info` includes:
```

Comment:
```suggestion
The `script info LABEL...` command lists details about all script definitions matching the given LABELs (see [run](./run) command for details about matching labels). The information provided by `script info` includes:
```
the link is malformed.

Additionally, I think the link text should be "script run" to not confuse with `terramate run`.